scvi/models/modules.py

```diff
@@ -19,10 +58,23 @@ class FCLayers(nn.Module):
     :param n_layers: The number of fully-connected hidden layers
     :param n_hidden: The number of nodes per hidden layer
     :param dropout_rate: Dropout rate to apply to each of the hidden layers
+    :param n_batch_norm: Number of categories to conditionally batch normalize to (1 is
+             equivalent to vanilla batch norm, 0 no batch norm)
+             See `n_instances` in ConditionalInstanceNormalization for more information
+    :param use_batch_norm: Deprecated, use n_batch_norm=0 instead
```

Comment:
Better to delete it rather than having a "deprecate" note...we're not at v 1.0 so better to keep the code simple than to maintain backward compatibility.